installing/installing_cloud_other/verify-installation-cloud-other.adoc

```diff
@@ -0,0 +1,27 @@
+:_content-type: ASSEMBLY
+[id="verify-installation-cloud-other"]
+= Verifying installation of RHACS Cloud Service on other platforms
+include::modules/common-attributes.adoc[]
+:context: verify-installation-cloud-other
+
+toc::[]
+
+[role="_abstract"]
+Provides steps to verify that {product-title-managed-short} is properly installed.
```

Comment:
Just curious, what is the purpose of this? Seems to say the same thing as the first sentence.

Reply:
I think the abstract is used in some place on the customer portal - could rewrite a little so it doesn't look odd